python/pyspark/ml/feature.py

```diff
@@ -200,9 +203,28 @@ def approxSimilarityJoin(self, datasetA, datasetB, threshold, distCol="distCol")
         return self._call_java("approxSimilarityJoin", datasetA, datasetB, threshold, distCol)
 
 
+class _BucketedRandomProjectionLSHParams():
+    """
+    Params for :py:class:`BucketedRandomProjectionLSH` and
+    :py:class:`BucketedRandomProjectionLSHModel`.
+    .. versionadded:: 3.0.0
```

Comment:
There should be an empty line between the description and  `.. versionadded:: ...` (or any directive)

```python
    """
    Params for :py:class:`BucketedRandomProjectionLSH` and
    :py:class:`BucketedRandomProjectionLSHModel`.


    .. versionadded:: 3.0.0
```

Otherwise such elements are not recognized as directives and incorrectly appended to the body.